pages/dkp/kommander/2.2/DKP-upgrade/Upgrade Kommander/index.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: layout.pug
+navigationTitle: Upgrade Kommander
+title: Upgrade Kommander
+menuWeight: 20
+excerpt: Steps to upgrade Kommander via CLI
+beta: false
+enterprise: true
+---
+ 
+This section automatically upgrades your Kommander cluster and all its Platform Applications to the latest versions. Kommander is installed on your Management cluster and upgrading it must be the first step you take when upgrading DKP, as it will prevent compatibility issues.
+
+<p class="message--note"><strong>NOTE: </strong>It is important that you upgrade Kommander before upgrading Konvoy and Kubernetes. Kommander 2.1 does not support Kubernetes 1.22, and upgrading Kubernetes to 1.22 will be part of the Konvoy upgrade.</p>
+
+## Prerequisites
+
+- [Download][download-binary] and install the latest DKP CLI binary on your computer.
+- Ensure you are on DKP version 2.1 or 2.1.1 and Kubernetes version 1.21.
+- If you have attached clusters, ensure they are on Kubernetes versions 1.19, 1.20 or 1.21. To upgrade your Kubernetes version, refer to [AKS][AKS], [AWS][AWS], [Azure][Azure], [EKS][EKS], [pre-provisioned][pre-provisioned] documentation. [enterprise type="inline" size="small"/]
```

Comment:
@makkes @jimmidyson  We want to allow n-2 k8s versions as attached clusters. So when we go to k8s 1.22, the attached cluster range should be 1.20 and 1.21.
In my opinion we should remove 1.19 from this statement, do you agree?
cc @gvasquezvargas